repo/packages/D/datadog/0/marathon.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "datadog-agent",
+    "container": {
+        "type": "DOCKER",
+        "docker": {
+            "image": "alq666/docker-dd-agent:mesos",
+            "parameters": [
+                {
+                    "key": "name", "value": "dd-agent"
+                },
+                {
+                    "key": "env", "value": "API_KEY={{datadog.api_key}}"
+                }
+            ],
+            "network": "BRIDGE",
+            "portMappings": [
+                { "hostPort": 8125, "containerPort": 8125, "protocol": "udp" },
```

Comment:
Could you please make the `hostPort` configurable for both of these?  Maybe you could default to the canonical values.